src/api/index.ts

```diff
@@ -68,6 +70,27 @@ export const start = async (chainId = 1): Promise<void> => {
     },
   });
 
+  server.auth.strategy("webhook_server", "basic", {
```

Comment:
@gulfamali Can you attach a sample curl request with the auth header here, just as an example for understanding

Reply:
```curl
curl --location 'http://localhost:3000/register-app' \
--header 'Content-Type: application/json' \
--header 'Authorization: Basic d2ViaG9vazpqZGVkZzM3ZDM4ZDg=' \
--data-raw '{
    "name": "Pacman",
    "email": "gulfam1@priori.capital",
    "webhookUrl": "http://localhost:3000/health-check",
    "webhookAuthKey": "authkey12"
}'
```

```js
/** 
* Here username is `webhook` and password is WEBHOOK_SERVER_KEY in .env. 
* Pass the generated string after concatenating username and password 
* separated by colon symbol and base64 encoded.
**/
btoa('webhook:jdedg37d38d8') => 'd2ViaG9vazpqZGVkZzM3ZDM4ZDg=';
```